doc/source/whatsnew/v0.18.2.txt

```diff
@@ -101,6 +101,7 @@ API changes
 ~~~~~~~~~~~
 
 
+- ``DataFrame.to_csv()`` has dropped the ``engine`` parameter (:issue:`13418`)
```

Comment:
No API changes like this in a minor release?  It has been deprecated for a long time though...do changes like these only go for major releases?